docs/source/daml/reference/packages.rst

```diff
@@ -0,0 +1,53 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Reference: DAML packages
+########################
+
+This page gives reference information on DAML package dependencies:
+
+.. contents:: :local:
+
+DAML archives
+*************
+
+When a DAML project is build with ``daml build``, build artifacts are generated in the hidden
+directory ``.daml/dist/`` relative to the project root directory. The main build artifact of a
+project is the `DAML archive`, recoginzed by the ``.dar`` file ending. DAML archives can be deployed
+on a ledger (see :ref:`deploy <daml-ref-deploy>`) or can be imported into other projects as a package dependency.
+
+Importing DAML archives
+***********************
+
+A DAML project can import DAML package dependencies. Note that currently there is no package
```

Comment:
Should that be `DAML archive`? We seem to be a bit inconsistent on whether we call it a package or an archive.

Reply:
I'm also not quite sure about what our terminology is.

Reply:
I go with archive for now